tests/before_script.sh

```diff
@@ -11,14 +11,6 @@ if [ "$TRAVIS_SECURE_ENV_VARS" = "true" ]; then
   composer config -g github-oauth.github.com $GH_OAUTH
 fi
 
-# Make a ramdisk for mysql (speed improvement)
-sudo mkdir /mnt/ramdisk
-sudo mount -t tmpfs -o size=1024m tmpfs /mnt/ramdisk
-sudo stop mysql
-sudo mv /var/lib/mysql /mnt/ramdisk
-sudo ln -s /mnt/ramdisk/mysql /var/lib/mysql
-sudo start mysql
-
```

Comment:
Do you observe a change in job execution duration ?

Reply:
Errors were more consistent and running times aren't too much different

Reply:
If time don't change a lot, OK to drop. However this shall not affect tests. OK to drop